src/ontology/edits/oeo-shared-axioms.omn

```diff
@@ -4028,9 +4028,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1485",
         rdfs:label "demand"@en
     
     SubClassOf: 
-        <http://purl.obolibrary.org/obo/BFO_0000017>,
-        OEO_00010121 some 
-            (OEO_00030022 or <http://purl.obolibrary.org/obo/BFO_0000030>)
```

Comment:
Is this axiom deleted on purpose? It is part of `demand`, not `energy service demand`

Reply:
<img width="815" alt="Bildschirmfoto 2023-09-14 um 14 07 06" src="https://github.com/OpenEnergyPlatform/ontology/assets/135724657/c1b8c993-e7c0-4683-af19-0ff7c73c003a">

Ah right, that probably happened when I removed the double axioms of energy service demand. I also removed the Subclass of the Ancestor demand, because it was there twice, but I recognised now, that the second one belongs to oeo-shared and should be moved in the according pull request for the shared-classes.